apps/meteor/ee/server/models/raw/LivechatUnit.ts

```diff
@@ -168,6 +168,11 @@ export class LivechatUnitRaw extends BaseRaw<IOmnichannelBusinessUnit> implement
 		return this.findOne(query, options);
 	}
 
+	async findUnitsByMonitorId(monitorId: string): Promise<FindCursor<IOmnichannelBusinessUnit>> {
+		const monitoredUnits = await LivechatUnitMonitors.findByMonitorId(monitorId).toArray();
+		return this.find({ _id: { $in: monitoredUnits.map((u) => u.unitId) } }, {});
```

Comment:
Avoid referencing models from within other models - In this case its best to make a simple `findByIds` and pass the list of unitIds directly (so wherever you called `findUnitsByMonitorId`, you call both `findByMonitorId` to get the units and then `findByIds`)

Reply:
I'm gonna remove this one anyways since it's no longer needed (ther was a need, but after building the UI the need dissapeared)

But yeah, this is a copy/paste of the one right under it.